modules/@apostrophecms/import-export-doc-type/index.js

```diff
@@ -27,5 +27,27 @@ module.exports = {
     }
 
     self.apos.doc.addContextOperation(operation);
+
+    // Add import operation for singleton doc types
+    if (self.options.singleton === true || !!self.options.singletonAuto) {
+      if (self.options.importExport?.import === false) {
+        return;
```

Comment:
Not sure it should return here, but like above build the `excludedTypes` (a specific one for import) on that? like done for export.

Reply:
removed code